### PR TITLE
Package properties rpc endpoint

### PR DESCRIPTION
**Pull request by chrisguida**
This PR implements the package properties endpoint

### CODE REVIEW

**Comment by elvece** [comment]
First merge `update/compat` PR and rebase onto integration after it is merged

File: compat/src/main.rs [hunk 1]
```diff
@@ -1,24 +1,126 @@
...
+            SubCommand::with_name("properties")
+                .arg(
+                    Arg::with_name("package-id")
+                        .help("The `id` field from the manifest file")
+                        .required(true),
+                )
                 .arg(
                     Arg::with_name("mountpoint")
-                        .help("The `mount` field from manifest.yaml")
+                        .help("The backups mount point")
```
Comment (elvece) [inline compat/src/main.rs:115]: update this description for properties

File: compat/src/main.rs [hunk 2]
```diff
@@ -1,24 +1,126 @@
...
                         .required(true),
                 )
                 .arg(
-                    Arg::with_name("spec")
-                        .help("The path to the config spec in the container")
+                    Arg::with_name("datapath")
+                        .help("The path to the data to be backed up in the container")
```
Comment (elvece) [inline compat/src/main.rs:120]: update this description for properties

**Comment by chrisguida** [comment]
Closed in favor of #462

**Pull request closed without merging**